chains/evm/contracts/test/pools/FastTransferTokenPool/FastTransferTokenPool.ccipReceive.t.sol

```diff
@@ -87,7 +87,7 @@ contract FastTransferTokenPool_ccipReceive_Test is FastTransferTokenPoolSetup {
   function test_ccipReceive_WithDifferentDecimals() public {
     uint8 sourceDecimals = 6; // USDC-like decimals
     uint8 destDecimals = IERC20Metadata(address(s_token)).decimals();
-    require(sourceDecimals != destDecimals, "Test requires different source and destination decimals");
+    assertTrue(sourceDecimals != destDecimals, "Test requires different source and destination decimals");
```

Comment:
The only change to code in this PR, as it was very minor and allows us to remove a rule override.